meetings/160217_organisers.md

```diff
@@ -12,3 +12,52 @@
 - Beyond the workshop
 - Report 2015
   - [draft text](../reporting/report_2015.md)
+
+###Present:
+Ben Laken, Jaša Čalogović (visiting from Hvar Observatory, University of Zagreb), Leon du Toit, Hugues Fontenelle, Karin Lagesen, Halfdan Rydbeck, Anne Fouilloux, Axel Rosén, Carrie Andrew, Lex Nederbragt (in charge of writing the minutes & summary)
+
+##Action points last meeting
+
+* recruiting helpers goes best face-to-face, remember to ask interested people to stay at the end of a workshop to talk to us
+* it was suggested we set up our own local checklists, e.g. pre-announce workshops to waiting list
+
+##Anchoring Carpentry@UiO, possible affiliate status
+
+* good news: the Science Library can use funds provided by USIT to pay for affiliate status for UiO
+* Karin and Lex, with Arnstein and Live, work towards getting the formalities in place
+
+##Past and upcoming workshops
+
+* Anne has set sent out a doodle poll to find dates for workshops for the rest of 2016
```

Comment:
Not sure we want the link to the doodle poll to be available to the whole world...